# Message describing the values from the inertial sensor
Header header

# raw data
float32 accX
float32 accY
float32 accZ
float32 gyrX
float32 gyrY

# corrected and filtered
float32 angleX
float32 angleY